### ISSUE TITLE
[CoE Starter Kit - BUG] Setup Flows not showing

### ISSUE THREAD

**Issue by guissilva** [microsoft/coe-starter-kit#6383]
### Does this bug already exist in our backlog?

- [X] I have checked and confirm this is a new bug.

### Describe the issue

Per the screenshot below, the setup flows are not showing for me (I've waited a couple hours and the page is still blank). 
Did I miss checking something?

![Bug_2023-08-18_Run_Setup_Flows](https://github.com/microsoft/coe-starter-kit/assets/44757356/f529569d-93d4-42b6-b8fb-adbf85010b7b)

### Expected Behavior

Show the flows that are supposed to run.

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.13

### What app or flow are you having the issue with?

CoE Setup and Upgrade Wizard

### What method are you using to get inventory and telemetry?

Cloud flows

### Steps To Reproduce

1. Configured all connections, according to the step: https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-core-components#create-connections
2. Imported the solution Core Components, according to the step: https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-core-components#import-the-core-components-solution
3. When importing solution used the connections configured on step 1.
4. After solution imported, opened the app CoE Setup and Upgrade Wizard, according to the step https://learn.microsoft.com/en-us/power-platform/guidance/coe/setup-core-components#open-the-setup-wizard-to-complete-the-configuration
5. Confirmed pre-requisites - all good
6. Configured communication methods - apparently configured correctly, although there is a UI bug (reported in another issue)
7. Configured mandatory seetings - used default for most fields, but as I'm doing this in 2 separate envs, one as prod, one as non-prod, setting up the environment parameter accordingly.
8. Configure inventory data source as Cloud Flows
9. Run setup flows - where the issue happens

**Comment by Jenefer-Monroe** [comment]
Are you a tenant in one of the specialty clouds like GCC?
Did you click refresh?

**Comment by Djuwal** [comment]
I'm having the same issue - get to the Run setup flows step and can't move forward. The required flows all activate, and complete just fine, but they never show up on the page, so the Next button never activates. I've clicked Refresh several times, even after the flows have completed. I am in the GCC and looking at the console in dev window of browser, I'm getting this error:

Uncaught (in promise) DOMException: Blocked a frame with origin "https://apps.gov.powerapps.us" from accessing a cross-origin frame. Dynamics runs in *.dynamics.com

**Comment by Jenefer-Monroe** [comment]
GCC (and other non-commercial tenants) are blocked right now via https://github.com/microsoft/coe-starter-kit/issues/6074.

From what I understand you will be unblocked at the end of the month.

**Comment by Djuwal** [comment]
Oof - okay, thanks for the info!

**Comment by guissilva** [comment]
@Jenefer-Monroe My tenant is not hosted in any special cloud, and I pressed refresh a few times.

And also it worked before in another environment, but we were using the Core Components release from July/23. We deleted that environment though.

**Comment by Jenefer-Monroe** [comment]
Can you please make sure all connections belong to you (or the user you desire to run the kit)

A recent change in behavior made connection references not get reassigned on upgrade. So if Person A installed to begin with and Person B upgrades, Person B used to get all the connection references auto-assigned and now does not.

We dont understand the consquences of this yet so can you go check the Default Solution > Connection References > click each one and assign to you (Person B). 

![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/240a02e4-1af4-4eee-91bb-329b44be1f66)

**Comment by guissilva** [comment]
Hi,

All connections are owned by Service Account A, the solution was installed with Service Account A and I am navigating through the app with Service Account A, but the connections themsevels are connected using a Service Account B.

We use A for all Power Apps deployments, development, etc, but B to connect to all services, because B has more permissions in the tenant.

Maybe is it impacting?

Thanks,
G

**Comment by Jenefer-Monroe** [comment]
You need to boot the setup qizard as whatever user you have owning those connetions as it will turn on a bunch of flows (including the ones you see missing here), and will not be able to do so if you are logged in as another user.

**Comment by guissilva** [comment]
Yes, I initiated the setup wizard using Service Account A, which had ownership of the connections. However, the connections themselves were actually connected to Service Account B.

But now, I just deleted the solution in the non-production, and have done everything from scratch with just Service Account B (installed it, owned the connections, connected to, and started the setup wizard) and it seems working. 😅🤷‍♂️

**Comment by guissilva** [comment]
Hi @Jenefer-Monroe 

Thanks for your help! As I mentioned, it worked well after I made everything with just the Service Account B in the non-production environment (ProductionEnvironment variable = No).

But now I'm trying to configure a production environment and the same issue is happening.
Again I'm just using 1 account for everything.

Can we reopen this issue?

Thanks,
Guilherme
